platforms/web/lib/conversion.ts

```diff
@@ -67,7 +67,7 @@ export async function richToPlain(richText: string) {
     return plainText;
 }
 
-export async function plainToRich(plainText: string) {
+export async function plainToRich(plainText: string, inMessageFormat = false) {
```

Comment:
nitpick: Would it be nicer to have the message format be the default so that the composer internals are only exposed if explicitly requested?

Reply:
I actually think that it's better to remove the default completely. I added it to avoid having to change all the function calls, but it opens the option of accidentally misusing the function. Will amend that in the next commit.